src/routes/profile-route/profile-fests-route.jsx

```diff
@@ -75,91 +119,87 @@ export const ProfileFestsRoute = () => {
+                                // needs to have {+ invitees.length - 5} to show how many invitees after 5
+                                return (
+                                  <Keyword
+                                    key={invitee.id}
+                                    className="h-31px space-x-2 space-y-2 bg-gray xs:space-x-2 xs:space-y-2 text-black text-center mr-2.5"
+                                    keyword={invitee.username}
+                                  />
+                                );
+                              })}
+                          {fest.invitees.length > 4 ? (
+                            <Keyword
+                              key={fest.id}
+                              className="h-31px space-x-2 space-y-2 bg-gray xs:space-x-2 xs:space-y-2 text-black text-center mr-2.5"
+                              keyword={`+ ${fest.invitees.length - 4} more`}
+                            />
+                          ) : (
+                            ""
+                          )}
+                        </div>
                       </div>
                     </div>
-                    {/* Button may need to be changed in future updates to include onClick functionality */}
-                    {/* Currently going and went states are based off endDate vs currentDate */}
-                    {dayjs().isAfter(items.endDate) ? (
-                      <Button
-                        variant={"secondary"}
-                        className="flex flex-row mb-5 h-10"
-                        size="sm"
-                        // onClick={""}
-                      >
-                        <img
-                          src={checkMarkWhite}
-                          alt="check mark"
-                          className="mr-2.5"
-                        />
-                        I went
-                      </Button>
-                    ) : (
+                    {(attendeeStatus || festDateInFuture) && (
                       <Button
-                        variant={"outline-secondary"}
-                        className="flex flex-row mb-5 h-10"
+                        variant={
+                          attendeeStatus ? "secondary" : "outline-secondary"
+                        }
+                        className={
+                          festDateInFuture
+                            ? "flex flex-row mb-5 h-10"
+                            : "flex flex-row mb-5 h-10 cursor-auto"
+                        }
                         size="sm"
-                        // onClick={""}
+                        onClick={
+                          festDateInFuture
+                            ? () => handleRSVPButtonClick(fest, attendeeStatus)
+                            : () => {}
+                        }
                       >
                         <img
-                          src={checkMark}
+                          src={attendeeStatus ? checkMarkWhite : checkMarkBlack}
```

Comment:
both checkMarkWhite and checkMarkBlack are undefined, these live under src/images and can be imported though!

Reply:
I'm showing both of these are imported on lines 15 & 16 above. Do you show that as well?